pysrc/bytewax/execution.py

```diff
@@ -22,13 +22,26 @@
 from bytewax.recovery import RecoveryConfig
 
 from .bytewax import (  # noqa: F401
+    run_main,
     cluster_main,
     EpochConfig,
     PeriodicEpochConfig,
-    run_main,
     TestingEpochConfig,
 )
 
+# Due to our package structure, we need to define __all__
+# in any submodule as pdoc will not find the documentation
+# for functions imported here, but defined in another submodule.
+# See https://pdoc3.github.io/pdoc/doc/pdoc/#what-objects-are-documented
+# for more information.
+__all__ = [
```

Comment:
My read of [the Python module docs for `__all__`](https://docs.python.org/3/tutorial/modules.html?highlight=__all__#importing-from-a-package) is that it is a manually maintained list of all public names this module contains, and if you define it, you're telling the module loader to not automatically determine public names. Thus it should contain `"spawn_cluster"` too, even though it is named as a "public" item.

Notice that it looks like the HTML docs now don't have spawn cluster.